### ISSUE TITLE
Decide on dynamic Icons

### ISSUE THREAD

**Issue by rjeantet** [TechLabs-Berlin/wt23-naturdoc#14]
Decide on dynamic icons:
- possibility to group / categorise remedies (DS) and create group icons (UX)?
- if so, implementation (WD)

**Comment by alj-b** [comment]
Brought this up a while ago on Slack, Luzie responded:

> we are planning to have only one icon, otherwise we would have to go through the whole database to caracterize categories

[Source](https://techlabs-community.slack.com/archives/C04N3KPJSCW/p1679414080040419?thread_ts=1678889916.167149&cid=C04N3KPJSCW)

**Comment by rjeantet** [comment]
Okay, thanks for posting it here Alojscha, I was not aware (or I forgot ^_^)
Well an easy alternative is also for me to randomize a set of icons, so at least it looks a little more visual.
If UX can give me 4-5 or how many they want, I can implement this.
Will reach out now to Eugenia and Luzie, see what they say.

**Comment by alj-b** [comment]
Yeah, that's what I initially mentioned as well - we'd either need to decide on how to categorise them or just randomise them.

The comment was part of another conversation so it's not surprising that it flew under your radar ;)

**Comment by hotmail030** [comment]
hejj so Eugenia and me talked today again and we decided against the icons. it's just too much noise 

![Bildschirmfoto vom 2023-03-23 10-01-33](https://user-images.githubusercontent.com/120741476/227153757-2a1a6015-d0b7-421e-ad07-2deeef4d51dc.png)
e just talked and we decided to not use any icons at all. its just too much noise
